docs/v0.4/getting-started/docker-for-windows.md

```diff
@@ -149,20 +159,17 @@ deployers.knative.projectriff.io      allowed   allowed
 Use the riff CLI to apply credentials to a container registry (if you plan on using a namespace other than `default` add the `--namespace` flag). Replace the ??? with your docker username.
 
 ```powershell
-DOCKER_ID=???
-```
-
-```powershell
-riff credential apply my-creds --docker-hub $DOCKER_ID --set-default-image-prefix
+riff credential apply my-creds --set-default-image-prefix --docker-hub ???
 ```
 
 You will be prompted to provide the password.
 
+> On Windows, a known issue prevents the password prompt from working in git bash. Use the Windows PowerShell or Command terminal instead.
```

Comment:
Do we have an issue for this?

The password can also be provided via stdin.

Reply:
https://github.com/projectriff/cli/issues/76